iocore/hostdb/P_RefCountCache.h

```diff
@@ -339,7 +338,7 @@ RefCountCachePartition<C>::count() const
 
 template <class C>
 void
-RefCountCachePartition<C>::copy(Vec<RefCountCacheHashEntry *> &items)
+RefCountCachePartition<C>::copy(std::vector<RefCountCacheHashEntry *> &items)
 {
   for (RefCountCachePartition<C>::iterator_type i = this->item_map.begin(); i != this->item_map.end(); ++i) {
```

Comment:
not that important.. but this can now be:

```c++
for (auto const &i : this->item_map) {
    RefCountCacheHashEntry *val = RefCountCacheHashEntry::alloc();
    val->set(i.item.get(), i.meta.key, i.meta.size, i.meta.expiry_time);
    items.push_back(val);
  }
```